pyneuphonic/_clone_voices.py

```diff
@@ -0,0 +1,50 @@
+from typing import List
+
+import httpx
+
+from ._endpoint import Endpoint
+
+
+class CloneVoices(Endpoint):
+    def post(
+        self, voice_name: str, voice_tags: List[str], voice_file_path: str
+    ) -> dict:
+        """
+        Clone a voice by uploading a file with specified name and tags.
```

Comment:
These need to be `numpy` docstring format. Check the docstring for some of the other classes.